openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,103 @@ The following subsections illustrate some of the authorization flows supported b
+    |  selects Credential |  (1b) Credential Offer (credential type)           |
+    |-------------------->|<---------------------------------------------------|
+    |                     |                                                    |
+    |                     |  (2) Obtains Issuer's Credential Issuer metadata   |
+    |                     |<-------------------------------------------------->|
+    |                     |                                                    |
+    |                     |  (3) Authorization Request                         |
+    |                     |      (type(s) of Credentials to be issued)         |
+    |                     |--------------------------------------------------->|
+    |                     |                                                    |
+    |           End-User Authentication / Consent                              |
+    |                     |                                                    |
+    |                     |  (4)   Authorization Response (code)               |
+    |                     |<---------------------------------------------------|
+    |                     |                                                    |
+    |                     |  (5) Token Request (code)                          |
+    |                     |--------------------------------------------------->|
+    |                     |      Token Response (access_token)                 |
+    |                     |<---------------------------------------------------|
+    |                     |                                                    |
+    |                     |  (6) Credential Request (access_token, proof(s))   |
+    |                     |--------------------------------------------------->|
+    |                     |      Credential Response                           |
+    |                     |      (Credential(s) OR Transaction ID)             |
+    |                     |<---------------------------------------------------|
 ~~~
 !---
 Figure: Issuance using Authorization Code Flow 
 
-(1) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. Note in this example, the Credential Issuer also provides the OAuth 2.0 Authorization Server. This specification enables deployments where the Credential Issuer API and the Authorization Server are different services, perhaps even provided by different entities.
+(1a) The Wallet-initiated flow begins as the End-User requests a Credential from a pre-configured list of the Credentials ready to be issued or, in alternative, as the End-User requires a Credential not available yet in the Wallet and the Wallet gives guidance to the End-User to get the Credential from a Credential Issuer.
```

Comment:
I am not sure how these two things are `in alternative`..? meaning, end-user is requesting a credential from a pre-configured list because it is not available in the wallet, yet, no?

Reply:
Technically they might be the same, from a UX perspective they are quite different in my opinion
a) the issuer starts within the wallet and chooses to request some pre-configred credentials
b) as part of OpenID4P request for a credential, the wallet tries to get one on the fly
This is also reflected by the different use case examples

Reply:
ah ok, so for the second part you are trying to give an example of an issuance during presentation. how about this:
```suggestion
(1a) The Wallet-initiated flow begins as the End-User requests a Credential from a pre-configured list of the Credentials ready to be issued or, in alternative, it can be issuance during presentation, where the Wallet gives guidance to the End-User to get the Credential from a Credential Issuer based on the information received in the presentation request.
```